src/observable/dom/AjaxObservable.ts

```diff
@@ -22,7 +22,7 @@ export interface AjaxRequest {
   responseType?: string;
 }
 
-const createXHRDefault = (): XMLHttpRequest => {
+function createXHRDefault() {
```

Comment:
Actually, type `XMLHTTPREQUEST` in here is return type of this function, so it can be still preserved even replacing it to this way of using standard function. (It's TypeScript syntax, as you mentioned)

Or either leave this as is and bind context when it's being used at [here](https://github.com/opsb/RxJS/blob/a2afe1e117359e92e89c26a0d0a2ced7390e1214/src/observable/dom/AjaxObservable.ts#L130) also can be way to go.